apps/core-app/src/components/customFields/RagequitToken.tsx

```diff
@@ -0,0 +1,55 @@
+import { useMemo } from 'react';
+import { RegisterOptions, useFormContext } from 'react-hook-form';
+import { toWholeUnits, handleBaseUnits } from '@daohaus/common-utilities';
+import { Buildable, Button, WrappedInput, Field } from '@daohaus/ui';
+
+import { useConnectedMembership, useDao } from '@daohaus/dao-context';
+
+export const RagequitToken = (props: Buildable<Field>) => {
+  const { id } = props;
+  const { setValue } = useFormContext();
+  const { dao } = useDao();
+  const { connectedMembership } = useConnectedMembership();
+
+  const daoTokenData = useMemo(() => {
+    if (!dao || !connectedMembership) return null;
+    return {
+      label:
+        id === 'sharesToBurn'
+          ? `Voting Tokens (${dao.shareTokenSymbol})`
+          : `Non-Voting Tokens (${dao.lootTokenSymbol})`,
+      maxAmount:
+        id === 'sharesToBurn'
+          ? connectedMembership.shares
+          : connectedMembership.loot,
```

Comment:
question about loot and shares -- can a member only have 1 or the other, or can they have both?

if both, just curious-- would we want to present that here?

Reply:
They can have both - this input is used in both instances - so we have it in the form twice - once for shares and once for loot.

```ts
       fields: [
          {
            id: 'sharesToBurn',
            // @ts-expect-error: cannot resolve type within formSegment using custom fields sent from the core-app
            type: 'ragequitToken',
          },
          // @ts-expect-error: cannot resolve type within formSegment using custom fields sent from the core-app
          { id: 'lootToBurn', type: 'ragequitToken' },
        ],
        ```